config/main.cpp

```diff
@@ -27,12 +27,23 @@
 
 #include <LXQt/SingleApplication>
 
+#include <QCommandLineParser>
 #include "main_window.h"
 
 int main(int argc, char *argv[])
 {
     LXQt::SingleApplication a(argc, argv);
 
+    QCommandLineParser parser;
+    parser.setApplicationDescription(QStringLiteral("LXQt Config Globalkeys "));
+    const QString VERINFO = LXQT_GLOBALKEYS_VERSION \
```

Comment:
@agaida I do agree with @palinek . Actually I wrote the same thing on lxqt-admin not knowing this one.

All our new  code should compile witth:
``` CMake
add_definitions(
    -DQT_USE_QSTRINGBUILDER
    -DQT_NO_CAST_FROM_ASCII
    -DQT_NO_CAST_TO_ASCII
    -DQT_NO_URL_CAST_FROM_STRING
    -DQT_NO_CAST_FROM_BYTEARRAY
)
```

Most of our code doesn't and some day it should be addressed. In the meantime, while there are more urgent tasks, we should not worsen the problem.